src/main/java/edu/harvard/iq/dataverse/FileDownloadServiceBean.java

```diff
@@ -455,10 +455,11 @@ public boolean requestAccess(Long fileId) {
     }    
     
     public void sendRequestFileAccessNotification(Dataset dataset, Long fileId, AuthenticatedUser requestor) {
-        permissionService.getUsersWithPermissionOn(Permission.ManageDatasetPermissions, dataset).stream().forEach((au) -> {
-            userNotificationService.sendNotification(au, new Timestamp(new Date().getTime()), UserNotification.Type.REQUESTFILEACCESS, fileId, null, requestor);
-        });
+        Stream<AuthenticatedUser> usersWithManageDsPerm = permissionService.getUsersWithPermissionOn(Permission.ManageDatasetPermissions, dataset).stream();
+        Stream<AuthenticatedUser> usersWithManageMinorDsPerm = permissionService.getUsersWithPermissionOn(Permission.ManageMinorDatasetPermissions, dataset).stream();
 
+        Stream.concat(usersWithManageDsPerm, usersWithManageMinorDsPerm).forEach((au) ->
+                userNotificationService.sendNotification(au, new Timestamp(new Date().getTime()), UserNotification.Type.REQUESTFILEACCESS, fileId, null, requestor));
```

Comment:
It will most probably send an email twice if some user have both `ManageDatasetPermissions` and `ManageMinorDatasetPermissions` permissions